dashboard/lib/widgets/task_icon.dart

```diff
@@ -51,6 +51,7 @@ class TaskIcon extends StatelessWidget {
     final String matchedName = qualifiedTask.task!.toLowerCase();
     final bool isWebTest = matchedName.contains('_web') || matchedName.contains('web_');
     final bool isToolTest = matchedName.contains('_tool') || matchedName.contains('tool_');
+    final bool isAndroidTest = matchedName.contains('_android') || matchedName.contains('_mokey');
```

Comment:
All the instances of `_android` should be gone, and I think all we have now are `_mokey` and `_pixel_7pro`.

Reply:
Maybe next time we do this we use "_highend" and "_lowend" or something similar so the builder names don't change.

Reply:
Changing the builder name is our current mechanism for starting a new series in SkiaPerf, which we want when we change the kind of device a benchmark runs on. We could investigate a new mechanism for that, and then we could leave the builder names the same.

Reply:
Or they could be different, but have some kind of stable way to identify them as Android from the name agnostic of the device hardware, so this doesn't need to be updated every time.